### PR TITLE
Update Gempackage category if it has no category yet

### PR DESCRIPTION
**Pull request by cveneziani**

### CODE REVIEW

File: app/services/gempackages/update_category_service.rb [hunk 1]
```diff
@@ -46,8 +46,12 @@ def gempackage
     end
 
     def must_update_category?
+      return false if has_category?
       gempackage.last_ruby_toolbox_check_at.nil? || gempackage.last_ruby_toolbox_check_at < 1.day.ago
     end
 
+    def has_category?
```
Comment (kdisneur) [inline app/services/gempackages/update_category_service.rb]: Shouldn't we move this method before `must_update_category?` (to follow alphabetical order)?

Comment (cveneziani) [inline app/services/gempackages/update_category_service.rb]: Sure. I'll do this.

**Comment by cveneziani** [comment]
Done.

**Comment by kdisneur** [comment]
:+1:

**Comment by alaibe** [comment]
:+1:

**Pull request merged**